### ISSUE TITLE
VPC/Subnet requirements not documented

### ISSUE THREAD

**Issue by matthewadams** [amplab/spark-ec2#90]
I don't believe that all of the requirements to use `spark-ec2` with an existing VPC & subnet are documented.  If I create a VPC & subnet, then use those to run `spark-ec2` as documented [here](https://github.com/amplab/spark-ec2#launching-a-cluster-in-a-vpc), I get an error ending with
```
Waiting for cluster to enter 'ssh-ready' state...........
Error:
 Failed to determine hostname of Instance:i-0909d1ed1af09cd09.
Please check that you provided --private-ips if necessary
```
More info on SO at http://stackoverflow.com/questions/42654336/how-do-i-resolve-failed-to-determine-hostname-of-instance-error-using-spark-ec

**Comment by yuj** [comment]
I am running into the same issue.  Have you got any update?

**Comment by matthewadams** [comment]
@yuj nope...it's been so long I can't remember what the solution or workaround was...sorry.
